model/AI/Vocabularies/EnergyConsumptionDescriptionUnitType

```diff
@@ -0,0 +1,21 @@
+SPDX-License-Identifier: Community-Spec-1.0
+
+# EnergyConsumptionDescriptionUnitType
+
+## Summary
+
+Specifies the unit of energy consumption
+
+## Description
+
+List different acceptable units of energy consumption. If the unit in which the energy consumption is recorded is not available here, please select other
+
+## Metadata
+
+- name: SafetyRiskAssessmentType
+
+## Entries
+
+- kW-H: Kilo Watt/ hour.
```

Comment:
"kW-H: Kilo Watt/ hour."
->
"kWh: Kilowatt-hour." (lowercase "h")

Note that NIST suggested "kW · h" or "kW h" but that's probably too difficult to deal with as an entry name. See https://nvlpubs.nist.gov/nistpubs/Legacy/SP/nistspecialpublication811e2008.pdf (Notes 2 of the section 6.1.5 at page 12).

We can just use a more common "kWh" (NIST also recognized this practice, but not recommended).